### PR TITLE
Created stub methods

### PR DESCRIPTION
**Pull request by jbell-97**
Started code in methods characteristic and mantissa.

### CODE REVIEW

File: assignmentPR/conversionF1.c [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// CS499-Group01-SiteScribe
+// Pull Request Assignment
+// 4th Sept 2018
+// Owner: Colton Nunley
+// Contributers: Joseph Remy, Julian Bell, and Tanner Massahos
+
+char number[] = "123.456";
+int c, n, d;
+
+// Returns the part of a float before the decimal (like a floor function)
```
Comment (tanman987) [inline assignmentPR/conversionF1.c]: These really are quite stubby

Comment (raptorace27) [reply assignmentPR/conversionF1.c]: Agreed, but lets give constructive feedback please.

Comment (tanman987) [reply assignmentPR/conversionF1.c]: let's*

Comment (raptorace27) [reply assignmentPR/conversionF1.c]: I should have also mentioned the lack of punctuation in the first comment...

**Pull request merged**